demo/App.svelte

```diff
@@ -36,9 +38,9 @@
   <Button filled>
     Button!!
   </Button>
-  <RadioGroup items={colorItems} color bind:value={colorItem} name="colors" />
+  <!-- <RadioGroup items={colorItems} color bind:value={colorItem} name="colors" />
   <RadioGroup items={items} bind:value={item} name="numbers" />
-  <CheckboxGroup items={items} max={1} name="numbers-check" />
+  <CheckboxGroup items={items} max={1} name="numbers-check" /> -->
```

Comment:
Why is this commented out?

Reply:
It was too long on the page and making me sick. This isn't even a proper demo page anyway, I just needed some space to showcase the components from the latest PRs. It's okay to leave this commented.